powersimdata/design/transmission/upgrade.py

```diff
@@ -197,12 +197,16 @@ def scale_congested_mesh_branches(
     upgrade_n=100,
     allow_list=None,
     deny_list=None,
-    quantile=0.95,
+    congestion_metric="quantile",
+    cost_metric="branches",
+    quantile=None,
     increment=1,
-    method="branches",
 ):
-    """Use a reference scenario as a baseline for branch scaling, and further
-    increment branch scaling based on observed congestion duals.
+    """Use a reference scenario to detect congested mesh branches (based on values of
+    the shadow price of congestion), and increase the capacity of a subset of them.
+    Branches are ranked by a ratio of (congestion metric) / (cost metric), and the top N
```

Comment:
N -> ``upgrade_n``?

Reply:
`the top N, where N is specified by \`\`upgrade_n\`\``?

Reply:
I like 'top N' for a narrative description, even though I agree it is clearer to indicate that it's the `upgrade_n` parameter that specifies.

Reply:
Done.